modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java

```diff
@@ -184,6 +184,9 @@
     public static final String DOTNET_FRAMEWORK = "targetFramework";
     public static final String DOTNET_FRAMEWORK_DESC = "The target .NET framework version.";
 
+    public static final String TEMPLATING_ENGINE = "templatingEngine";
+    public static final String TEMPLATING_ENGINE_DESC = "The templating engine plugin to use";
```

Comment:
Minor suggestion similar to https://github.com/OpenAPITools/openapi-generator/pull/2657/files#r275162756